src/components/form/ImageForm.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import axiosInstance from "../api/axios";
+import "./ImageForm.css";
+
+const ImageForm = ({ closeModal }) => {
+  const [userDetails, setUserDetails] = useState({ title: "", image: null });
+  const [message, setMessage] = useState("")
+
+  const handleChange = (e) => {
+    e.preventDefault();
```

Comment:
Why prevent default?

Reply:
And what is the default action?

Reply:
I have removed it, its not a form, now corrected in the new ticket.